Fruitties/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.10-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-bin.zip
```

Comment:
Is there a reason to downgrade the Gradle version?

Reply:
It was a warning during build. I reset it to 8.10, rebuilt and all worked as expected.